### ISSUE TITLE
OpenCDMS does not have support for hydrological modules

### ISSUE THREAD

**Issue by hjedwards** [opencdms/risk#4]
### Description

OpenCDMS is predominantly focused on supporting CDMS. Of the current focus systems, only the Meteorological, Climatological and Hydrological Database Management System (MCH) has support of hydrological parameters and processes.

Without support for hydrological processes and data management the OpenCDMS project will not achieve its broader aims.

### Mitigation

Prioritise support for MCH involvement in the project in 2021.

**Comment by Steve-Palmer** [comment]
Agreed that this is an issue which needs attention. One of the requirements is to define the data management needed in the data model to meet the hydrology requirements. One aspect is to work out how to store observational data needed by the hydrology domain in the same way as traditional climatology variables. There is need to add data types for e.g. staff gauges for stream height, v-notch weirs, float or pressure systems and stream-flow measurements. All these need appropriate metadata, especially the related discharge curves with calibration dates and evidence. Even within MCH, much of this metadata is not recorded and is assumed to be held in other systems (possibly paper-based).  

It is likely that adding storage of the basic observation data for hydrology will not require significant changes to the OpenCDMS data model, but the addition of rating curves and the data behind these may require some extensions. Documentation of data requirements for hydrology are documented in WMO #49 and WMO #168. 

Note that the hydrology-specific extensions to a data model originally conceived for climatology will provide valuable experience for adding other "whole-earth" observation systems, including for atmospheric chemistry and pollution. As for hydrology,  each observation system needs its own history and calibration. In addition, other observation systems are likely to need rating systems to derive useful parameters for applications.

In summary, priority actions are to engage with the MCH design team and others to identify the data and metadata requirements for the hydrology specific observation systems, and to trial implementation of these within the data model being used as the development base for the OpenCDMS data model, with the aim of the hydrology products being consistent regardless of whether the observation data is obtained from an MCH implementation of from the trial model of OpenCDMS.
